cmd/sink/logs/logs.go

```diff
@@ -32,6 +34,9 @@ var (
 	loggingCmSelector = corev1.ConfigMapKeySelector{
 		Key: "logging-configmap-name",
 	}
+	containerName = map[string]string{
```

Comment:
Why is this a `map` and not just a constant string?

Reply:
This was left over from some previous messing around. Fixed now